tests/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.1'
+
+services:
+
+  db:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: example
+      POSTGRES_DB: demo
+
+  nri-postgresql:
+    image: golang:1.16
+    container_name: nri_postgresql
+    working_dir: /code
+    depends_on:
+      - db
+    volumes:
+      - ../:/code
+    entrypoint: go run /code/cmd/nri-postgresql/main.go
```

Comment:
I like this approach instead of creating the image , just wandering if a fail on the go run (because a bad compilation) would be something explicit in the test logs but i guess it would.